src/jenkins_job_insight/main.py

```
@@ -2485,6 +2486,73 @@ async def _load_effective_failure(
     return failure, result_data
 
 
+@app.get("/results/{job_id}/issue-prompt")
+async def get_issue_prompt(
+    job_id: str,
+    settings: Settings = Depends(get_settings),
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)